thesheriff/application/raid/end_raid.py

```diff
@@ -11,13 +11,13 @@ class EndRaid:
     def __init__(self, raid_repository: RaidRepository):
         self.raid_repository = raid_repository
 
-    def execute(self, raid_id: int, raid_score: int):
-        """execute is the actual action of the Raid rating use case.
+    def execute(self, raid_id: int, raid_score: float):
```

Comment:
Maybe a `EndRaidRequest `class holding those two values? Just in order to have same "expected" object at execute method.

Worth it?

Reply:
makes sense! I'm adding it and will update the PR.